api/src/certification/configuration/infrastructure/repositories/learning-content-repository.js

```diff
@@ -0,0 +1,56 @@
+import _ from 'lodash';
+
+import { LOCALE } from '../../../../shared/domain/constants.js';
+import * as areaRepository from '../../../../shared/infrastructure/repositories/area-repository.js';
+import * as challengeRepository from '../../../../shared/infrastructure/repositories/challenge-repository.js';
+import * as competenceRepository from '../../../../shared/infrastructure/repositories/competence-repository.js';
+import * as skillRepository from '../../../../shared/infrastructure/repositories/skill-repository.js';
+import * as thematicRepository from '../../../../shared/infrastructure/repositories/thematic-repository.js';
+import * as tubeRepository from '../../../../shared/infrastructure/repositories/tube-repository.js';
+
+export async function getConsolidatedFrameworkLearningContentByChallengeIds({ challengeIds }) {
```

Comment:
Je trouve ce nom très technique, je propose de simplifier 😅 

```suggestion
export async function getConsolidatedFrameworkChallenges({ challengeIds }) {
```

Et en vrai apres lecture de la fonction ca serait pas plutot ?

```suggestion
export async function getFrameworkReferential({ challengeIds }) {
```